### PR TITLE
leader election log message validation

### PR DESCRIPTION
**Pull request by miyadav**
This is to automate OCP-33455

Validated on AWS - https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/152260/console
Azure - https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/150151/console

### CODE REVIEW

**Comment by miyadav** [comment]
@sunzhaohua2  @jhou1  , PTAL when you get time

File: features/machine/machine.feature [hunk 1]
```diff
@@ -303,4 +303,28 @@ Feature: Machine features testing
...
+      | machineset-controller   | |
+      | nodelink-controller     | |
+
+   Then I repeat the following steps for each :id in cb.containers:
+   """
+   When I run the :logs admin command with:
+      | resource_name | <%= pod.name %> |
+      | c             | #{cb.id}        |
+    Then the output should contain:
+      | attempting to acquire leader lease  openshift-machine-api/cluster-api-provider |
```
Comment (pruan-rht) [inline features/machine/machine.feature:328]: @miyadav should there be two spaces betweeen ```leader lease  openshift-machine-api/cluster-api-provider```

File: features/machine/machine.feature [hunk 2]
```diff
@@ -303,4 +303,28 @@ Feature: Machine features testing
...
+
+    Given a pod becomes ready with labels:
+      | api=clusterapi, k8s-app=controller |
+
+    And I saved following keys to list in :containers clipboard:
+      | machine-controller      | |
+      | machineset-controller   | |
+      | nodelink-controller     | |
+
+   Then I repeat the following steps for each :id in cb.containers:
```
Comment (sunzhaohua2) [inline features/machine/machine.feature:322]: need to align left.

File: features/machine/machine.feature (hunk 1 above)
Comment (jhou1) [reply features/machine/machine.feature:328]: Nice, catch.

Comment (miyadav) [reply features/machine/machine.feature:328]: yeah it is like that  :) ..

File: features/machine/machine.feature (hunk 2 above)
Comment (miyadav) [reply features/machine/machine.feature]: done

File: features/machine/machine.feature (hunk 1 above)
Comment (pruan-rht) [reply features/machine/machine.feature:328]: @miyadav ok, that's probably a bug...nitpicking kind...very low priority

**Review by pruan-rht — COMMENTED** [review]
/lgtm

**Pull request merged**